tests/ci_scripts/test_bump_version.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+import pathlib
+
+import pytest
+
+from ci_scripts.bump_version import (
+    bump_version_from_news_files,
+    news_ext_to_version_type,
+)
+
+
+@pytest.mark.parametrize(
+    "files,expected_version",
+    (
+        ((pathlib.Path("blah.feature"), pathlib.Path("blah.misc"), pathlib.Path("blah.bugfix")), "0.1.0"),
```

Comment:
Should we also test .removal?

Reply:
`.removal` doesn't affect the version. That is tested in another test.